workbench-core/environments/src/services/environmentService.test.ts

```diff
@@ -284,7 +284,7 @@ describe('EnvironmentService', () => {
 
       // OPERATE
       const actualResponse = await envService.listEnvironments(
-        { roles: ['Admin'], id: 'owner-123' },
+        { roles: ['ITAdmin'], id: 'owner-123' },
```

Comment:
Just confirming that this is correct. Although `environments` is under `workbench-core`, it is SWB concept, not MAF at this point.

Reply:
Yes, I had to make this change in order for tests to pass.